src/resources/assets/scss/customs/_list.scss

```diff
@@ -135,10 +135,16 @@
     tr td:first-child {
       display: flex;
       align-items: center;
-      padding-top: 1rem !important;
-      padding-bottom: 1rem !important;
       padding-left: 0.6rem !important;
     }
+    
+    &[data-responsive="1"] {

```

Comment:
Could we also account for the `data-responsive` missing entirely here? Maybe we can use a `.not([data-reponsive])` or something like that, to DEFAULT to responsive? 

I'm thinking of projects that have overridden their `list.blade.php` already. If we merge this like this, because they're missing `data-reponsive` in their blade files, their tables wouldn't be responsive any more.

Reply:
@tabacitu that makes total sense!
I "inverted" the logic here, so now this will always work, even if the user has overridden their `list.blade.php` 👌